### ISSUE TITLE
App crashes/doesn't open when clicking on the number dialer thing

### ISSUE THREAD

**Issue by kirk781** [SimpleMobileTools/Simple-Dialer#591]
I updated the app to the latest version on F-Droid and since then whenever, I click on the dialer thing to make a call via inputting a number, it hangs/crashes. I went back to the previous version via F-Droid and boom it works.  I am on Android 12 with most Google services disabled [though that didn't hamper any app from your set to crash in the past].

I have a vanilla Nokia running in a half broken condition that runs Android 13. I could probably check on that phone if it is Android causing a bug.

**Comment by tibbi** [comment]
what app version are you using?

**Comment by kirk781** [comment]
Currently, I am on 5.17.5 , the last stable version for me. The version 5.17.6 was buggy for me. F-Droid published a new update  5.17.7 which I installed and it seems to be running smoothly again. Just to be on the clear side that it wasn't Android's doing , I installed the buggy version again to see if it was replicating again and yes, the problem was still there. Eithercase, the new update seems to have put matters in the right place, so err, thank you for the new version.
